content/components/action-list.mdx

```diff
@@ -0,0 +1,221 @@
+        <Box as="p">Trailing visual and trailing text can display auxiliary information. They're placed at the right of the item, and can denote status, keyboard shortcuts, or be used to set expectations about what the action does.</Box>
+        <Box as="p">Note these side visuals don't have dedicated interaction targets.</Box>
+        <Box as="p">Use an `arrow-right` octicon in menus to indicate the action will open more options, such as in a nested context. Use an `edit` octicon to indicate the item is going to be edited after clicking it.</Box>
+        <Box as="p">Custom trailing elements are supported, such as counters, labels, and other custom visuals that may help identify the item.</Box>
+        <Box as="p">When using a trailing text for displaying keyboard shortcuts, always confirm the characters match with the user's operating system. For example, to indicate a bold action in a Markdown toolbar, use "Ctrl+B" on Linux and Windows, and "⌘B" on Mac. <a href="https://support.apple.com/en-us/HT201236">See reference for Mac keyboard glyphs</a>.</Box>
+    </div>
+    <div>
+        <Flex justifyContent="center">
+            <img width="464" alt="Item dividers" src="https://user-images.githubusercontent.com/293280/124041789-6c5c4f00-d9bc-11eb-8638-e8d0f0bca2e0.png" />
+        </Flex>     
+        <Flex justifyContent="center">&nbsp;</Flex>
+    </div>
+    <div>
+        <Heading fontSize={3}>Item dividers</Heading>
+        <Flex as="p">Item dividers allow users to parse heavier amounts of information. They're placed between items and are useful in complex lists, particularly when descriptions or multi-line text is present.</Flex>
+        <Flex as="p">When considering whether to use item dividers, make sure they truly make the presented information easier to parse, instead of only increasing visual clutter.</Flex>
+        <Flex as="p">When using item dividers, increasing the action list item size may also help with legibility.</Flex>
+    </div>
+    <div>
+        <Flex justifyContent="center">
+            <img width="464" alt="Action list item in danger variation" src="https://user-images.githubusercontent.com/293280/124039646-33ba7680-d9b8-11eb-965c-f5f10e76cdbd.png" />
+        </Flex>        
+    </div>
+    <div>
+        <Heading fontSize={3}>Danger</Heading>
+        <Box as="p">An action list item can have a special "danger" style, to be used in cases that require extra attention from the user.</Box>
+        <Box as="p">For destructive or irremediable actions, show a confirmation dialog for extra friction. If the action is not destructive, present the user a way to undo the action instead of asking for confirmation. <a href="https://alistapart.com/article/neveruseawarning/">Never use a warning when you mean undo</a>.</Box>
+        <Box as="p">Place danger items at the end of the list.</Box>
+    </div>
+</Grid>
+
+## Application
+
+### In overlays
+
```

Comment:
The components listed below can't have a link until the page exists. Can we comment this section out, or remove the links before we publish? @vdepizzol

Reply:
Fixed!